packages/components/src/Banner/Banner.module.css

```diff
@@ -51,19 +76,24 @@
 
   @media (--small-screens-and-below) {
     width: 100%;
-    margin-top: var(--space-small);
+    padding-left: var(--space-larger);
```

Comment:
Margin-top no longer needed because we've been using `gap` for some time, the padding-left aligns the action with the text content.

![image](https://github.com/user-attachments/assets/22cde920-8596-432e-b2cb-99a20d18750e)